examples/clipping.rs

```diff
@@ -165,8 +175,8 @@ fn orthographic_projection(width: u32, height: u32) -> [f32; 16] {
     #[cfg_attr(rustfmt, rustfmt_skip)]
     [
         2.0 / width as f32, 0.0, 0.0, 0.0,
-        0.0, 2.0 / height as f32, 0.0, 0.0,
+        0.0, -2.0 / height as f32, 0.0, 0.0,
         0.0, 0.0, 1.0, 0.0,
-        -1.0, -1.0, 0.0, 1.0,
+        -1.0, 1.0, 0.0, 1.0,
```

Comment:
Changing the coordinate system is a breaking change, so it should probably be mentioned somewhere.

Reply:
This is a change in `wgpu`, which `wgpu_glyph` extends. But you are right, we should mention it in the `CHANGELOG`, nevertheless.